civismlext/preprocessing.py

```diff
@@ -148,14 +149,25 @@ def __init__(self,
         self.fill_value = fill_value
         self.dataframe_output = dataframe_output
 
+    def _flag_nulls(self, X, cols_to_drop):
+        null_cols = [col for col in X if
+                     X[col].isnull().all() and col not in cols_to_drop]
```

Comment:
If you flip the order of these checks, then you could save a little bit of run time. If users request to drop a column and the `col not in cols_to_drop` comes first, then you'll avoid having to check the column for nulls.